spec/logr/entry_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'spec_helper'
+
+describe Logr::Entry do
+  let(:logger) { double('logger') }
+  let(:entry) { Logr::Entry.new(logger) }
+
+  describe '#event' do
+    let(:entry_event) { entry.event('test_event', context_1_key: 'context_1_value') }
+
+    it 'should add the event with the correct name' do
+      expect(entry_event.instance_variable_get(:@event).name).to eq('test_event')
+    end
+
+    it 'should add the event with the correct context' do
+      expect(entry_event.instance_variable_get(:@event).context).to include(context_1_key: 'context_1_value')
```

Comment:
I would not test instance variables as well. It might be a good idea to expose a getter method for `@event` and test against that. If we want to later create different formatters they might need direct access to the event.

As `event` is already taken we have to resort to `get_event` or something similar.